src/services/color_picker/color_picker.ts

```diff
@@ -57,10 +57,16 @@ export const useColorStopsState = (
   return [colorStops, updateColorStops, addColor];
 };
 
-export const useColorPickerState = (initialColor = '') => {
+export type _SetColorMethod = (
```

Comment:
Probably want to make this a more global export then, because consumers will have the same issue. Use `Eui` prefix and export from `index`